xbmc/windows/GUIWindowLoginScreen.cpp

```diff
@@ -246,12 +246,12 @@ bool CGUIWindowLoginScreen::OnPopupMenu(int iItem)
     return true;
   }
   
-  if (!g_passwordManager.IsMasterLockUnlocked(true))
```

Comment:
But the early return is not the best solution here, because even though the user clicks on the [x] after the right button click it will still prompt for the master lock password. This should only happen when you press "Edit profile", I assume.